axelrod/tests/strategies/test_defector.py

```diff
@@ -44,9 +44,18 @@ def test_strategy(self):
         # Starts by defecting.
         self.first_play_test(D)
         self.second_play_test(D, D, D, D)
-        # Test if strategy tries to trick opponent.
```

Comment:
i think i got them.  i tried to cover all the behavior more explicitly here.  that was why i separated out the lists.  i was trying to show where the deviations would occur (or mirror it and fail to occur) without making the reader scan a list of random c's and d's.